lib/classes/PluginManager.js

```diff
@@ -75,20 +76,15 @@ class PluginManager {
   loadServicePlugins(servicePlugs) {
     const servicePlugins = Array.isArray(servicePlugs) ? servicePlugs : [];
 
+    // eslint-disable-next-line no-underscore-dangle
+    module.paths = Module._nodeModulePaths(process.cwd());
+
     // we want to load plugins installed locally in the service
     if (this.serverless && this.serverless.config && this.serverless.config.servicePath) {
-      module.paths.unshift(
-        path.join(this.serverless.config.servicePath, 'node_modules'),
-        path.join(this.serverless.config.servicePath, '.serverless_plugins')
-        );
+      module.paths.unshift(path.join(this.serverless.config.servicePath, '.serverless_plugins'));
     }
 
     this.loadPlugins(servicePlugins);
-
-    // restore module paths
-    if (this.serverless && this.serverless.config && this.serverless.config.servicePath) {
-      module.paths.shift();
```

Comment:
My understanding is that `module.paths` only have influence for loading within that same module. And therefore my assumption is that restoring the module paths would only be useful if `require()` was used after the paths has been altered, e.g to load a module which would be in the serverless install dir node_modules (or parents) and not in the `process.cwd`. Also I wouldn't know how to test that it's been restored.

Moreover, since all the PluginManger deps are required before it is instantiated and the core plugins are loaded with absolute paths, I think it would be more explicit and future proof to move the `module.paths` reassignment in the `PluginManager.constructor` instead of `loadServicePlugins`. What do you think?